src/biogeophys/WaterStateType.F90

```diff
@@ -150,6 +157,14 @@ subroutine InitAllocate(this, bounds, tracer_vars)
     call AllocateVar1d(var = this%dynbal_baseline_ice_col, name = 'dynbal_baseline_ice_col', &
          container = tracer_vars, &
          bounds = bounds, subgrid_level = subgrid_level_column)
+    !excess ice vars
```

Comment:
Indent comment to same level as code.

Reply:
Indented code to the same level as code above.

Reply:
Perfect.